### ISSUE TITLE
Database doesn't open after entering password

### ISSUE THREAD

**Issue by Dinoperson** [Kunzisoft/KeePassDX#1768]
**Describe the bug**

If you want to unlock a database, you land on the start screen

**To Reproduce**

1. Click on 'Vorhandene Datenbank öffnen'
2. Choose a .kdbx file
3. Enter correct password
4. Click on 'Entsperren'
5. See that you're on the start screen

**Expected behavior**

I guess that you can view your just unlocked database

**KeePass Database**

 - Created with: Arch Linux KeePassXC 2.7.6[e.g Windows KeePass 2.42]
 - Version: ?
 - Location: Remote file retrieved with nextcloud app
 - File provider (`content://` URI): https://apps.nextcloud.com/apps/android_nextcloud_app
 - Size: 29,5 KB
 - Contains attachment: No

**KeePassDX:**

 - Version: 4.0.5
 - Build: Free
 - Language: German

**Android:**

 - Device: Galaxy S21 FE 5G
 - Version: 14

**Additional context**

//

**Comment by J-Jamet** [comment]
I haven't encountered this problem, but since you're using the NextCloud application, check that the URI of the kdbx file is accessible locally.

**Comment by Dinoperson** [comment]
> I haven't encountered this problem, but since you're using the NextCloud application, check that the URI of the kdbx file is accessible locally.

We tried some other things and it seems like I can't use Argon2id (KDBX 4) or Argon2d (KDBX 4) as key derivation function (Schlüssel-Ableitungsfunktion in German). So it does work now with AES-KDF (KDBX 4). My partner doesn't run into this issue at all though and he doesn't use a Samsung device. So my best guess is that the issue appears because of my OS.
I've just tried saving the file locally before trying to open it with the App and it still doesn't work.
